doc/source/getting-started.rst

```diff
@@ -0,0 +1,194 @@
+.. _getting-started:
+
+*********************
+Getting Started Guide
+*********************
+
+The following guide will step through an example of developing and testing a
+new Ansible role. After reading this guide, you should be familiar with the
+basics of how to use Molecule and what it can offer.
+
+.. note::
+
+    In order to complete this guide by hand, you will need to additionally
+    install `Docker`_.
+
+.. _Docker: https://docs.docker.com/
+
+Creating a new role
+-------------------
+
+Molecule uses `Ansible Galaxy`_ under the hood to generate conventional role
+layouts. If you've ever worked with roles before, you'll be right at home. If
+not, please review the `Role directory structure`_ guide to see what each
+folder is responsible for.
+
+.. _Ansible Galaxy: https://docs.ansible.com/ansible/latest/reference_appendices/galaxy.html#the-command-line-tool
```

Comment:
Could you please try this?
```suggestion
.. _Ansible Galaxy: https://docs.ansible.com/ansible/latest/reference_appendices\
                    /galaxy.html#the-command-line-tool
```
(I didn't test it, though. I hope to make links fit into the common line-length limit)
But it's minor, disregard it in any event of failure or if you think that the link shouldn't be broken into pieces.